### PR TITLE
4 get api reviews

### PR DESCRIPTION
**Pull request by fuzzyJess**
Finished tasks for GET api/reviews

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -29,38 +29,59 @@ describe("GET requests", () => {
...
+                .then(({ body }) => {
+                    expect(body.msg).toBe("Path not found");
+                })
+        });
+    })
+    describe("status 400 errors", () => {
+        test("invalid review_id, responds with 'Review ID not found' message", () => {
+            return request(app)
+            .get("/api/reviews/95")
+            .expect(400)
```
Comment (alexdevdavis) [inline __tests__/app.test.js:80]: this should return a 404

File: models/reviews.models.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const db = require("../db/connection");
+
+exports.selectReview = (id) => {
+    
+    return db.query(`SELECT * FROM reviews WHERE review_id = $1`,  [id])
+    .then((data) => {
+        const review = data.rows[0];
+        if (!review) {
+            return Promise.reject({
+                status: 400,
```
Comment (alexdevdavis) [inline models/reviews.models.js:10]: a 400 error might be if someone tried to find review_id: "sausage" ... ideally you'll account for both 404 and 400

**Review by alexdevdavis — APPROVED** [review]
Happy to approve this on the basis that you ensure both 404 and 400 errors are accounted for in your solution. On the whole, code is looking good 'n' clean. Nice one

File: __tests__/app.test.js (hunk 1 above)
Comment (fuzzyJess) [reply __tests__/app.test.js:80]: Ah yes - will do, thanks Alex

File: models/reviews.models.js (hunk 2 above)
Comment (fuzzyJess) [reply models/reviews.models.js:10]: Right - have updated tests & app to deal with this.

**Pull request merged**